documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1210,15 +1210,15 @@ Emitting the tombstone event (the default behavior) allows Kafka to completely d
 |_empty string_
 | A semi-colon list of regular expressions that match fully-qualified tables and columns to map a primary key. +
 Each item (regular expression) must match the `<fully-qualified table>:<a comma-separated list of columns>` representing the custom key. +
-Fully-qualified tables could be defined as `DB_NAME.TABLE_NAME` or `SCHEMA_NAME.TABLE_NAME`, depending on the specific connector.
+Fully-qualified tables could be defined as _pdbName_._schemaName_._tableName_.
```

Comment:
IIRC, the connector is already scoped for a single PDB, right? Does it then actually make sense to require the PDB name for these FQNs?

Reply:
Perhaps using `pdbName` here is misleading but rather it should probably be `catalogName` instead since ColumnMappers are based on `TableId` and oracle uses the format:
```
catalogName.schemaName.tableName
```
What do you think?

Reply:
That's a good question.  If my understanding of the code is accurate, this boils down to the call in `ColumnMappers#fullyQualifiedColumnName` where the call to `ColumnId#toString` takes the string representation of the `TableId` and appends `.<columnName>`.  

Perhaps the best of both worlds might be to introduce some logic that when we return the expected string comparison value if a schema is part of the `TableId`, we omit the catalog portion from the return value; where-as if a schema is not part of the `TableId` and a catalog is that we use catalog with the `table.column` format.

This would preserve the catalog behavior with MySQL where there is no schema and would otherwise work for the other relational connectors that specify a schema value, some with and some without the catalog value.

@jpechane Do you have any suggestions or thoughts here?